cache/src/commonMain/kotlin/com/dropbox/kmp/external/cache3/Cache.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 André Claßen
```

Comment:
Is it an issue these are under a copyright of the author or should we move it to Dropbox?

Reply:
Nope! P-Counsel signed off on this. As long as contributor signs our CLA which states the code is theirs and not stolen + we can use with our license. The copyright is similar with what we do with Googlers (see Multicaster)

Reply:
There is an open question to Legal of this being a port of the Guava cache implementation and not retaining their original copyright. I noticed the LocalCache implementation does that, but this is also derived from there (perhaps indirectly, but it's apparent from the JavaDocs) so I would it would need to retain the "Guava Authors" attribution.

Reply:
I used the sources from :
https://github.com/dropbox/Store/tree/main/cache/src/main/java/com/dropbox/android/external/cache3
Where no copyright is in the source files.
So i added the copyright from the guava sources , as it's not my intention at all to claim the guava authors work.
I'll change this in the other files too, i just missed it.

Reply:
Thanks, I figured that was the case (and pointed out to our legal counsel that our previous version didn't retain the license in the source).